WIP 3.0 Classes/Mage - Reworked.md

```diff
@@ -0,0 +1,1152 @@
+
+### Ability Score Improvement
+When you reach 4th level, and again at 8th, 12th, 16th, and 19th level, you can increase one ability score of your choice by 2. or you can increase two ability scores of your choice by 1. As normal, you can't increase an ability score above 20 using this feature.
+
+### Spell Mastery
+At 18th level, you have achieved such mastery over certain spells that you can cast them at will. Choose a 1st-level mage spell and a 2nd-level mage spell that are in your spell&shy;book. You can cast those spells at their lowest level without expending a spell slot when you have them prepared. If you want to cast either spell at a higher level, you must expend a spell slot as normal.
+
+<div class='footnote'>PART 1 | CLASSES</div>
+
+\pagebreakNum
+
+You can spend 8 hours in study, and change one or both of the spells you chose for different spells of the same level.
+
+### Signature Spells
+When you reach 20th level, you gain mastery over <br> two powerful spells and can cast them with little effort. <br> Choose two 3rd-level mage spells in your spellbook as your signature spells. You always have these spells prepared, they don't count against the number of spells you have prepared, and you can cast each of them once at 3rd level without expending a spell slot. When you do so, you can't do so again until you finish a short or long rest.
+
+If you want to cast either spell at a higher level, you must expend a spell slot as normal.
+
+## Magical Studies
+Mages are spellcasters who favor magics involving the cardinal elements of the universe. Students gifted with a keen intellect and unwavering discipline may walk the path of the mage. The magic available to mages is both great and dangerous, and thus is revealed only to the most devoted practitioners.
+
+### Study of Arcane
+Mages of the arcane are diviners of secrets, balancing the webb and flow of incredible mystic energies. These practi-tioners push their magical knowledge to its very limits. Those who master this craft are capable to manipulate spells on a much more primal level.
+
+#### Bonus Cantrip
+*2nd-level Arcane feature* <br>
+When you choose this magical study at 2nd level, you learn the *prestidigitation* cantrip. If you already know this cantrip, you learn a different wizard cantrip of your choice. The cantrip doesn't count against your number of cantrips known.
+
+\columnbreak
+
+#### Arcane Charges
+*2nd-level Arcane feature* <br>
+You gain the ability to conserve and enhance your mage abilities beyond the norm. Whenever you cast a mage spell of 1st level or higher, gain an arcane charge of the equivalent level up to 5th level. Spells slots higher than 5th give a 5th level arcane charge. These charges last until replaced by a higher level charge, used, or 1 minute has passed.
+
+You may expend an arcane charge as a bonus action to grant yourself one of the following effects. This effect lasts until the end of your next turn.
```

Comment:
There's two ways written below this for how to use an arcane charge, and each of them are different. 

Maybe change the formatting here so that the leading text doesn't specify the particular kind of action, or so that it specifies that each use has its own action, and then the bullet-pointed entries can start with "as a bonus action, gain a bonus", "as a reaction, gain a bonus"?

Reply:
You may expend an arcane charge to grant yourself one of the following effects. This effect lasts until the end of your next turn.
- As a bonus action, you can add a bonus equal to the arcane charge's level to Attack and Damage rolls for your next Mage Spell.
- As a bonus action or reaction, you can add a bonus equal to the arcane charge's level to Saving Throws against Magic or Magical Effects.

Reply:
Oh, that one was intended so that it could be a bonus action or reaction? It's a lot clearer like this, so that is a good thing. The original wording made it sound like, "instead of a bonus action, you use a reaction", entirely omitting a bonus action for it. 🤔 

Hence why it felt like, these two uses have very little action overlap. 

Is it a typical thing, for something to be usable with either a bonus action or a reaction?

Reply:
Not at all.